lnwallet/channel.go

```diff
@@ -2282,7 +2282,7 @@ func NewBreachRetribution(chanState *channeldb.OpenChannel, stateNum uint64,
 
 	// Query the on-disk revocation log for the snapshot which was recorded
 	// at this particular state num.
-	revokedSnapshot, err := chanState.FindPreviousState(stateNum)
+	_, revokedSnapshot, err := chanState.FindPreviousState(stateNum)
```

Comment:
IIRC, this also needs to be updated to use either of the types depending on if this was from the old or new revocation log?

Reply:
Yep changed in the following commit.